components/03-organisms/menu/utility-nav/_utility-nav.scss

```diff
@@ -0,0 +1,89 @@
+@use '../../../00-tokens/index.scss' as base;
+@use '../../../01-atoms/lists/list';
+
+.utility-bar {
+  --link-color-base: var(--color-text);
+  --utility-bar-gap: var(--size-spacing-7);
+
+  // Color schemes
+  [data-component-theme='white'] &,
+  [data-component-theme='gray-100'] & {
+    --link-color-hover: var(--color-blue-medium);
+    --link-styles-no-underline-bg-color: var(--color-blue-medium);
+  }
+
+  @include base.body-s;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--utility-bar-gap);
+
+  @media (max-width: base.$break-mobile-max) {
+    --utility-bar-gap: var(--size-spacing-6);
+
+    flex-direction: column;
+  }
+
+  input {
+    // @TODO: Temporary styles until the search is 'officially' created
+    background: transparent;
+    border: none;
+    color: inherit;
+    font-weight: var(--font-weights-mallory-500);
+
+    &::placeholder {
+      color: inherit;
+
+      @media (max-width: base.$break-mobile-max) {
+        text-align: center;
+      }
+    }
+
+    &:hover {
+      &::placeholder {
+        color: var(--link-color-hover);
+      }
+    }
+
+    &:focus {
+      &::placeholder {
+        color: var(--color-gray-300);
+        font-style: italic;
+      }
+    }
+
+    @media (max-width: base.$break-mobile-max) {
+      border: 2px solid;
```

Comment:
I know these are tmp styles so ignore if needed. I'm guessing these styles will be captured as tokens once you get to the forms/form inputs components?

Reply:
I went ahead and applied/created appropriate tokens for these. Might as well do it now so that it's easier to copy/paste to the actual search component later.